governance/votes/Executive vote - March 13, 2023.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Template - [Executive Vote] Emergency PSM Changes - March 13, 2023
+summary: Introduce a PSM Breaker, a new exception to GSM delay, which can instantly disable further borrowing from individual PSMs once an executive vote has passed.
+date: 2023-03-13T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Emergency PSM Changes - March 13, 2023
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an emergency executive proposal into the voting system, on behalf of the Risk Core Unit. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **additions** will be made to the Maker Protocol:
+- Add a PSM breaker, [described in this post](https://forum.makerdao.com/t/out-of-schedule-executive-proposal-to-implement-psm-breaker/20162), which will allow Governance to bypass the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) when setting individual [Peg Stability Modules](https://manual.makerdao.com/module-index/module-psm) to a [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) of 0.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **16 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Add PSM Breaker
+
+If this executive vote passes, based on this [emergency recommendation](https://forum.makerdao.com/t/out-of-schedule-executive-proposal-to-implement-psm-breaker/20162), a Peg Stability Module breaker will be added for all active PSMs (listed below) that would allow Governance to individually to set their respective [Debt Ceilings](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) to 0 with an executive vote.
```

Comment:
```suggestion
If this executive vote passes, based on this [emergency recommendation](https://forum.makerdao.com/t/out-of-schedule-executive-proposal-to-implement-psm-breaker/20162), a Peg Stability Module breaker will be added that would allow Governance to individually to set the [Debt Ceilings](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) of any enabled PSM to 0 with an executive vote.
```